roles/edpm_nova/templates/nova_compute.json.j2

```diff
@@ -10,6 +10,9 @@
     },
     "volumes": [
         "/var/lib/openstack/config/nova:/var/lib/kolla/config_files:ro",
+	"/var/lib/openstack/certs/nova/tls.crt:/etc/pki/nova/server-cert.pem:ro",
+	"/var/lib/openstack/certs/nova/tls.key:/etc/pki/nova/private/server-key.pem:ro",
+	"/var/lib/openstack/cacerts/nova/TLSCABundleFile:/etc/pki/CA/cacert.pem:ro",
```

Comment:
because we are bindmounting the certs directly instead of having kolla copy them we will need to ensure the permission on the files are correct as noted above.

if we were using kolla_start to copy the file then we would not need to do this.

Reply:
done